packages/firestore/src/protos/google/firestore/v1/firestore.proto

```diff
@@ -17,6 +17,7 @@ syntax = "proto3";
 package google.firestore.v1;
 
 import "google/api/annotations.proto";
+import "google/firestore/v1/aggregation_result.proto";
```

Comment:
nit: move this import of `aggregation_result.proto` into its alphabetically-sorted location.